### PR TITLE
MISUV-4449 : Move Confidence Level property to config - calc component

### PR DESCRIPTION
**Pull request by AshokGitAccount**
### Description
Please include a brief summary of the change you've made and the issue that it fixes.

Add a link to the relevant story in Jira
[SASS-CHANGE-ME](ADD URL HERE)

##### Before Reviewing
- [x]  Have you pulled the branch down?
- [x]  Have you assigned yourself to the PR?
- [x]  Have you moved the task to “in review” on JIRA?
- [ ]  Have you checked to ensure all dependencies are up to date?
- [ ]  Have you checked to ensure its been rebased against the current version of main?

##### Whilst Reviewing
- [x]  Have you run the tests?
- [x]  Have you run the journey tests?
- [ ]  Have you looked at the JIRA story to make sure all Acceptance Criteria has been met?

##### After Reviewing
- [ ]  Have you checked for merge conflicts or any changes in the current main that may affect the current pull request? i.e. does it need another rebase?
- [ ]  Have you checked to make sure there are no builds in the pipeline before you merge?
- [ ]  Have you moved the task to “in pipeline” on Jira?

##### Before creating PR
- [ ]  Have you run the tests?
- [ ]  Have you run the journey tests? (where applicable)
- [ ]  Have you addressed warnings where appropriate?
- [ ]  Have you rebased against the current version of main?
- [ ]  Have you checked code coverage isn’t lower than previously?

##### After PRs been raised
- [ ]  Have you checked the PR Builder passes?

### CODE REVIEW

File: it/helpers/AuthStubs.scala [hunk 1]
```diff
@@ -73,7 +73,7 @@ trait AuthStubs {
       Json.obj("allEnrolments" -> enrolments)
   }
 
-  def authorised(response: JsObject = successfulAuthResponse(Some(Individual), Some(ConfidenceLevel.L200),mtditEnrolment,ninoEnrolment)): StubMapping = {
+  def authorised(response: JsObject = successfulAuthResponse(Some(Individual), Some(ConfidenceLevel.L250),mtditEnrolment,ninoEnrolment)): StubMapping = {
```
Comment (avelx) [inline it/helpers/AuthStubs.scala:76]: I would believe you need to extract this from config?

File: it/helpers/AuthStubs.scala [hunk 2]
```diff
@@ -96,7 +96,7 @@ trait AuthStubs {
       .willReturn(
         aResponse()
           .withStatus(OK)
-          .withBody(successfulAuthResponse(Some(Individual), Some(ConfidenceLevel.L200), otherEnrolment).toString())
+          .withBody(successfulAuthResponse(Some(Individual), Some(ConfidenceLevel.L250), otherEnrolment).toString())
```
Comment (avelx) [inline it/helpers/AuthStubs.scala:99]: also to be extracted from config

File: conf/application.conf [hunk 3]
```diff
@@ -126,6 +126,7 @@ microservice {
     auth {
       host = localhost
       port = 8500
+      confidenceLevel = 250
```
Comment (avelx) [inline conf/application.conf:129]: there is already confidenceLevel in the config

File: it/helpers/AuthStubs.scala (hunk 1 above)
Comment (AshokGitAccount) [reply it/helpers/AuthStubs.scala]: Done

File: it/helpers/AuthStubs.scala (hunk 2 above)
Comment (AshokGitAccount) [reply it/helpers/AuthStubs.scala]: Done

File: conf/application.conf (hunk 3 above)
Comment (AshokGitAccount) [reply conf/application.conf:126]: Done

Comment (avelx) [reply conf/application.conf:126]: this must be 200, as we are not lifting CL to 250 in this iteration

**Review by avelx — CHANGES_REQUESTED** [review]
changes as per comments

File: test/testUtils/TestSuite.scala [hunk 4]
```diff
@@ -73,14 +73,13 @@ trait TestSuite extends AnyWordSpec with MockFactory with BeforeAndAfterEach wit
...
   def mockAuth(enrolments: Enrolments = individualEnrolments) = {
-
     (mockAuthConnector.authorise(_: Predicate, _: Retrieval[_])(_: HeaderCarrier, _: ExecutionContext))
       .expects(*, Retrievals.affinityGroup, *, *)
       .returning(Future.successful(Some(AffinityGroup.Individual)))
 
     (mockAuthConnector.authorise(_: Predicate, _: Retrieval[_])(_: HeaderCarrier, _: ExecutionContext))
       .expects(*, Retrievals.allEnrolments and Retrievals.confidenceLevel, *, *)
-      .returning(Future.successful(enrolments and ConfidenceLevel.L200))
+      .returning(Future.successful(enrolments and ConfidenceLevel.L250))
```
Comment (avelx) [inline test/testUtils/TestSuite.scala:82]: must be extracted from config ..

**Review by avelx — APPROVED** [review]
looks good

**Review by avelx — APPROVED** [review]
approved

**Pull request merged**